cli/tests/auth.bats

```diff
@@ -15,6 +15,15 @@ load test_support.bash
 
 setup_file() {
   common_file_setup
+  export FLOX_FEATURES_USE_CATALOG=true
+  export  _FLOX_USE_CATALOG_MOCK="${BATS_FILE_TMPDIR}/empty-responses.json"
+  echo "[]" > "$_FLOX_USE_CATALOG_MOCK"
```

Comment:
Late thought after ticket refinement: we could commit this file to `tests/catalog_responses/empty-responses.json` to re-use this.